### ISSUE TITLE
Accessibility Audit: Improper label of article landmarks on Listings Page

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#959]
**ENVIRONMENT**
macOS 11.6 | Chrome Version 95.0.4638.69 (Official Build) (x86_64) | Voiceover
Windows 10 Enterprise | Chrome Version 96.0.4664.45 (Official Build) (64-bit) | JAWS
Windows 10 Enterprise | Firefox Version 92.0.2 (64-bit) | JAWS

**ISSUE**

Turn on the Screen Reader, and go to the [Listings](https://homeconnect.detroitmi.gov/listings) page. As you can see from the attached video, article landmarks in the page don't have a unique label. As a result, the screen reader announces multiple landmarks on the page only as article. This will be confusing if the user navigates using a screen reader. Ideally, the same type of landmarks in a page should have specific labels for easier navigation of the user.

https://user-images.githubusercontent.com/394908/148604375-0cd86196-6c24-4320-b4b4-8563d0513441.mov

**Comment by ColinBuyck** [comment]
No longer valid from LH audit
